src/main/java/loqor/ait/core/commands/RiftChunkCommand.java

```diff
@@ -65,11 +65,8 @@ private static int set(CommandContext<ServerCommandSource> context) {
 
         Text message;
 
-        if (!RiftChunkManager.isRiftChunk(targetBlockPos)) {
-            message = Text.translatable("command.ait.riftchunk.cannotsetlevel"); // This chunk is not a rift chunk, so
-            // you can't
-            // get the
-            // artron levels of it
+        if (!RiftChunkManager.isRiftChunk(source.getWorld(), targetBlockPos)) {
+            message = Text.translatable("command.ait.riftchunk.cannotsetlevel");
         } else {
```

Comment:
instead of an else couldnt you send the message earlier
i find it easier to read imo

Reply:
the if was there before me :sob:

Reply:
true that